torch/cuda/graphs.py

```diff
@@ -0,0 +1,89 @@
+import ctypes
+import gc
+import torch
+
+from ._utils import _dummy_type
+
+
+if not hasattr(torch._C, '_CudaStreamBase'):
+    # Define dummy base classes
+    torch._C.__dict__['CUDAGraph'] = _dummy_type('CUDAGraph')
+    torch._C.__dict__['graph_pool_handle'] = _dummy_type('graph_pool_handle')
```

Comment:
What's this?

Reply:
Cargo culted from [streams.py](https://github.com/pytorch/pytorch/blob/master/torch/cuda/streams.py#L7-L12) when i split graphs.py into its own file. Maybe it helps `import torch` succeed for CPU-only builds or something. I don't know if it's needed or correct in the new graphs.py. Is there someone I should ask? If you don't know I can dig further.